resources/ory/charts/hydra/templates/deployment.yaml

```diff
@@ -22,8 +22,6 @@ spec:
     matchLabels:
       app.kubernetes.io/name: {{ include "hydra.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
-  strategy:
-    {{- toYaml .Values.deployment.strategy | nindent 4 }}
```

Comment:
Why are we deleting this? Shouldn't that stay and become exposed in values.yaml?

Reply:
This should stay and is configured by us in the values.yaml